docker-compose.yml

```diff
@@ -10,11 +10,11 @@ services:
         build:
             context: .
             dockerfile: Dockerfile
-        image: flecsi-project:developing
+        # command: 
+        image: flecsi-project
+        user: stellar:stellar
         container_name: flecsi-stellar
         volumes:
             - ./spack:/home/stellar/src/spack
             - ./scripts/vimrc:/home/stellar/.vimrc
-            - ./scripts:/scripts/
```

Comment:
I think the changes inside container will not reflected on the host if you remove this.